src/manifest.js

```diff
@@ -261,22 +261,28 @@ export const addPropertiesToMaster = (master, uri) => {
   }
 
   forEachMediaGroup(master, (properties, mediaType, groupKey, labelKey) => {
-    if (!properties.playlists ||
-        !properties.playlists.length ||
-        properties.playlists[0].uri) {
-      return;
+    const groupId = `placeholder-uri-${mediaType}-${groupKey}-${labelKey}`;
+
+    if (!properties.playlists || !properties.playlists.length) {
+      properties.playlists = [Object.assign({}, properties)];
     }
 
-    // Set up phony URIs for the media group playlists since playlists are referenced by
-    // their URIs throughout VHS, but some formats (e.g., DASH) don't have external URIs
-    const phonyUri = `placeholder-uri-${mediaType}-${groupKey}-${labelKey}`;
-    const id = createPlaylistID(0, phonyUri);
+    properties.playlists.forEach(function(p, i) {
+      const id = createPlaylistID(i, groupId);
+
+      p.uri = p.uri || id;
```

Comment:
is `p.uri` expected to be the `groupId` (what used to be called `phonyUri`)?
Should `uri` continue to be set to `groupId`?

Reply:
Now that we can have more than 1 playlist in a group the groupId alone would conflict

Reply:
Wonder how big of an issue this is, since, it's potentially breaking.

Reply:
I guess we could keep playlist at index 0 as `groupId`, and use all others as `id`